dart/dynamics/MetaSkeleton.hpp

```diff
@@ -71,11 +71,21 @@ class MetaSkeleton : public common::Subject
   /// Default destructor
   virtual ~MetaSkeleton() = default;
 
+  /// Creates an identical clone of this MetaSkeleton
+  virtual MetaSkeletonPtr cloneMetaSkeleton(
+      const std::string& cloneName) const = 0;
+  // TODO: In DART7, rename this to clone() and change the current
+  // Skeleton::clone() to override it. Then consider adding
+  // Skeleton::cloneSkeleton().
```

Comment:
Super nitpick: We don't need to consider adding `Skeleont::cloneSkeleton()`, because it's already been added.